provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -2272,6 +2276,9 @@ spec:
             capabilities:
               add:
                 - NET_ADMIN
+          ports:
```

Comment:
The port can also be conditional, as it will only be pushing information if opflex metrics are enabled.

Reply:
Please mark this comment as resolved since you have addressed it.